### PR TITLE
Improve readme

### LINKED ISSUE #6: Improve README.md
- Add some badges.
    Ref: https://github.com/PyCQA/pylint/blame/master/README.rst#L5-L25
    build status can be Circle CI badge https://circleci.com/docs/2.0/status-badges/ requires #5
- Add expected output in command line examples as specification.

### PR DESCRIPTION
**Pull request by YukieK**
This PR resolves #6.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,12 +80,16 @@ Delete a study record:
 geekhours done rm date course_name
 ```
 
+## Command Examples
+
+[geekhours.md](https://github.com/YukieK/GeekHours/blob/master/geekhours.md)
```
Comment (mitsutaka) [inline README.md]: You don't need to use full path URLs.

File: README.md [hunk 2]
```diff
@@ -78,12 +80,16 @@ Delete a study record:
...
+
 ## Specifications
 
-[SPEC.md](SPEC.md)
+[SPEC.md](https://github.com/YukieK/GeekHours/blob/master/SPEC.md)
```
Comment (mitsutaka) [inline README.md]: ditto

File: README.md [hunk 3]
```diff
@@ -78,12 +80,16 @@ Delete a study record:
...
 
 ## License
 
 This project is licensed under the MIT License.
 
-[LICENSE](LICENSE)
+[LICENSE](https://github.com/YukieK/GeekHours/blob/master/LICENSE)
```
Comment (mitsutaka) [inline README.md]: ditto

File: geekhours.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+```
+$ geekhours course list
+[
+    {
+        'Python',
+        'English',
+        'Math'
+    }
+]
+```
```
Comment (mitsutaka) [inline geekhours.md]:
```json
[
    "Python",
    "English",
    "Math"
]
```

File: geekhours.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+        'course': 'English',
+        'duration': '1h'
+    },{
+        'id': 3,
+        'date': '20190302',
+        'course': 'Math'
+        'duration': '1h'
+    }
+]
+```
```
Comment (mitsutaka) [inline geekhours.md]: The single quote is invalid in JSON. Use double quote instead.

File: geekhours.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+## Delete a record of study time
+
+```
+$ geekhours done rm 20190302 Math
+Removed record.
+```
+
+## Delete course
```
Comment (mitsutaka) [inline geekhours.md]: Delete a course

File: geekhours.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
+# Geek Hours Command Examples
+
+## Help messages
```
Comment (mitsutaka) [inline geekhours.md]: Usage

File: geekhours.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+```
+$ geekhours -h
+usage: geekhours [-h] {course,done} ...
+
+Geekhours is a management tool for the study time.
```
Comment (mitsutaka) [inline geekhours.md]:
```
geekhours is a simple study time management tool.
```
You can use the same sentence of the README.md

File: geekhours.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+optional arguments:
+  -h, --help     show this help message and exit
```
Comment (mitsutaka) [inline geekhours.md]: remove "and exit" because of self-evident.

File: geekhours.md (hunk 5 above)
Comment (YukieK) [reply geekhours.md]: Fixed.

File: geekhours.md (hunk 6 above)
Comment (YukieK) [reply geekhours.md]: Fixed.

File: geekhours.md (hunk 7 above)
Comment (YukieK) [reply geekhours.md]: Fixed.

File: geekhours.md (hunk 8 above)
Comment (YukieK) [reply geekhours.md]: Fixed.

File: geekhours.md (hunk 9 above)
Comment (YukieK) [reply geekhours.md]: Fixed.

File: geekhours.md (hunk 4 above)
Comment (YukieK) [reply geekhours.md]: Fixed.

File: geekhours.md [hunk 10]
```diff
@@ -0,0 +1,95 @@
+# Geek Hours Command Examples
+
+## Usage
+```
```
Comment (mitsutaka) [inline geekhours.md]:
```
```console
```

File: geekhours.md [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+  geekhours done rm date course_name
+
+  {course,done}  Sub commands.
+    course       Manipulate course.
+    done         Manipulate done.
+```
+
+## Add courses
+
+```
```
Comment (mitsutaka) [inline geekhours.md]:
```
```console
```

File: geekhours.md [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+$ geekhours course add Python English Math
+Add 'Python' in course.
+Add 'English' in course.
+Add 'Math' in course.
+```
+
+## List courses
+
+```
```
Comment (mitsutaka) [inline geekhours.md]:
```
```console
```

File: geekhours.md [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+[
+    "Python",
+    "English",
+    "Math"
+]
+```
+
+## Add a record of study time
+
+```
```
Comment (mitsutaka) [inline geekhours.md]:
```
```console
```

File: geekhours.md [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+$ geekhours done add English -d 20190223
+Add '20190223 English 1h' in donelist.
+
+$ geekhours done add Math
+Add '20190302 Math 1h' in donelist.
+```
+
+## List records of study time
+
+```
```
Comment (mitsutaka) [inline geekhours.md]:
```
```console
```

File: geekhours.md [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+    },{
+        "id": 2,
+        "date": "20190223",
+        "course": "English",
+        "duration": "1h"
+    },{
+        "id': 3,
+        "date": "20190302",
+        "course": "Math",
+        "duration": "1h",
```
Comment (mitsutaka) [inline geekhours.md]:
```console
"duration": "1h"
```

File: geekhours.md [hunk 16]
```diff
@@ -0,0 +1,95 @@
...
+    }
+]
+```
+
+## Delete a record of study time
+
+```
```
Comment (mitsutaka) [inline geekhours.md]:
```console
```console
```

File: geekhours.md [hunk 17]
```diff
@@ -0,0 +1,95 @@
...
+$ geekhours done rm 20190302 Math
+Removed record.
+```
+
+## Delete a course
+
+```
```
Comment (mitsutaka) [inline geekhours.md]:
```console
```console
```

File: geekhours.md (hunk 15 above)
Comment (YukieK) [reply geekhours.md]: Fixed.

**Pull request merged**